tests/integration/security/rbac/grouplist/testdata/istio-group-list-rbac-rules.yaml.tmpl

```diff
@@ -0,0 +1,97 @@
+# istio-group-list-rbac-rules.yaml to enforce access control based on
+# groups and list claims for both http and tcp services using Istio RBAC rules.
+
+# For service a, allow no one to access it.
+# This will also result a default tcp rule for service a that denies all access.
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-http
+spec:
+  rules:
+    - methods: ["GET"]
+      constraints:
+        - key: "destination.labels[app]"
+          values: ["a"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: AuthorizationPolicy
+metadata:
+  name: authz-policy-access-a-http
+spec:
+  allow:
+    - subjects:
+        - not_names: ["allUsers"]
+      roleRef:
+        kind: ServiceRole
+        name: "access-a-http"
+---
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "require-jwt-for-b"
+spec:
+  targets:
+  - name: b
+  origins:
+  - jwt:
+      issuer: "testing@secure.istio.io"
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.1/security/tools/jwt/samples/jwks.json"
+  principalBinding: USE_ORIGIN
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: b-viewer
+spec:
+  rules:
+  - services: ["b.{{ .Namespace }}.svc.cluster.local"]
+    methods: ["GET"]
+---
+# Require a JWT with valid group claim to access service b
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
```

Comment:
Please only use `AuthorizationPolicy` in the new tests.

Reply:
Same as my previous reply: this set of tests (groupv1) is to cover the v1alpha1 version.